test_scripts/API/VehicleData/GetVehicleData/005_GetVD_versioning.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TO ADD !!!
+-- Use case: TO ADD !!!
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [GetVehicleData] As a mobile app wants to send a request to get the details of the vehicle data
+--
+-- Description:
+-- In case:
+-- mobile application sends valid GetVehicleData to SDL and this request is allowed by Policies
+-- SDL must:
+-- 1) Transfer this request to HMI
+-- 2) After successful response from hmi
+--    respond SUCCESS, success:true and parameter value received from HMI to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 3
```

Comment:
@YarikMamykin Why don't we use highest possible version? 
E.g. `fuelRange` applicable since="5.0" So lets register app with version `4.5`

Reply:
@dboltovskyi , fixed in 0c3f6df4ce34a698429a81f8a386c026c5389490